### ISSUE TITLE
unable to navigate to other messages from one card to another 

### ISSUE THREAD

**Issue by Wajeed-msft** [MicrosoftDocs/msteams-docs#4727]
Copied bug from: https://github.com/microsoft/AdaptiveCards/issues/6781 

**Details from original post:**
### Target Application

Microsoft Teams

### Application Operating System

Windows

### Schema Version

v1.2

### Problem Description

NVDA: When there are multiple cards for any response unable to navigate to other messages from one card to another using arrow keys
JAWS: Able to move from one message to another with arrow keys up/down
Note: As this accessibity feature is for the people who cannot see there should be a mechanism where user understand multiple messages are there to see. Because User dont know how many messages are there on the scren to view and Tab key is moving to next element but not to next message. Hence user dont know when to use arrow keys.

### Expected Outcome

When there are multiple cards for any response user should able to navigate to other messages from one card to another using arrow keys

### Actual Outcome

When there are multiple cards for any response user unable to navigate to other messages from one card to another using arrow keys

### Card JSON

```JSON
<msg><Adaptive>{
  "type": "AdaptiveCard",
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.2",
  "body": [
    {
      "type": "TextBlock",
      "text": "Here are the top results I found in Knowledge.",
      "wrap": true
    }
  ]
}</Adaptive></msg>{
  VendorRegistrationCarousel
}<msg><Adaptive>{
  "type": "AdaptiveCard",
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.2",
  "body": [
    {
      "type": "TextBlock",
      "text": "You can also perform an advanced search with filters.",
      "wrap": true
    }
  ],
  "actions": [
    {
      "type": "Action.OpenUrl",
      "title": "Perform advanced search",
      "url": "{AdavanceSearchUrl}"
    }
  ]
}</Adaptive></msg>
```

### Repro Steps

_No response_---
#### Document Details

⚠ *Do not edit this section.It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3fb4394d-25b1-8752-3996-fe3a2934732d
* Version Independent ID: 6d391381-89d0-117c-27cc-00043bd26a36
* Content: [Cards reference - Teams](https://docs.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/cards-reference)
* Content Source: [msteams-platform/task-modules-and-cards/cards/cards-reference.md](https://github.com/MicrosoftDocs/msteams-docs/blob/master/msteams-platform/task-modules-and-cards/cards/cards-reference.md)
* Product: **msteams**
* GitHub Login: @surbhigupta12
* Microsoft Alias: **surbhigupta12**

**Comment by Meghana-MSFT** [comment]
We are looking into this issue. We will update you

**Comment by Meghana-MSFT** [comment]
We tested this using NVDA, and whenever bot sends a carousal of adaptive cards, it just says you received a card message. is this what you are experiencing?

By multiple cards for any response you mean carousal of adaptive cards right?

**Comment by v-chetsh** [comment]
@prajwalman - Could you please provide your input for above query?

**Comment by prajwalmanju** [comment]
Hello Team,

we have 3 card response for single utterance, in such cases, we are unable to navigate from one card to another card using arrow keys while focusing on the card. Please find the attachment for the response.
Note : we are facing this issue while using NVDA screen reader.

<img width="416" alt="Multiple card response" src="https://user-images.githubusercontent.com/93922557/145722581-44557436-95d2-4b7e-b7bb-04aade21901d.PNG">

Thanks!

**Comment by Meghana-MSFT** [comment]
We are able to repro this issue. Attaching the videos of behavior with and without NVDA for your reference. We are raising a bug for the same.

With NVDA :

https://user-images.githubusercontent.com/81138249/146397297-9fa6e214-ab14-4fcb-9f55-3283b76ea3b3.mp4


Without NVDA :

https://user-images.githubusercontent.com/81138249/146397360-6ccc978e-f01f-4d9d-bcdf-c93281d17418.mp4

**Comment by Nivedipa-MSFT** [comment]
@prajwalmanju - I'm unable to reproduce this issue. Could you please test it on your end and confirm whether it's still occurring?
